modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -210,6 +210,18 @@ The block format is the default format for all newly-created databases as long a
 For more information on the block format, see xref:database-internals/store-formats.adoc[Store formats].
 ====
 
+[NOTE]
+====
+In Neo4j 2025.01, a new dual-named option `--from-pagecache, --max-off-heap-memory=<size>` replaces the `--from-pagecache=<size>` option, which was used to specify how much cache to allocate when reading the source.
+
+The new option determines the maximum amount of off-heap memory available during the copy process for reading and writing.
```

Comment:
```suggestion
The new option determines the maximum amount of off-heap memory available during the copy process for reading and writing,  instead of specifying how much cache to allocate when reading the source.
```